test/e2e/persistent_volumes.go

```diff
@@ -694,7 +694,7 @@ var _ = framework.KubeDescribe("PersistentVolumes", func() {
 
 		// Attach a persistent disk to a pod using a PVC.
 		// Delete the PVC and then the pod.  Expect the pod to succeed in unmounting and detaching PD on delete.
-		It("should test that deleting a PVC before the pod does not cause pod deletion to fail on PD detach", func() {
+		It("should test that deleting a PVC before the pod does not cause pod deletion to fail on PD detach [Volume][Serial][Flaky]", func() {
```

Comment:
I don't think there is a "serial flaky" tests suite. So AFAIK these will continue to run as part of Flaky. But these tests aren't just flaky, they're just broken ([consistently red](https://k8s-testgrid.appspot.com/google-gce#gce-flaky)). That should be addressed in a follow up.